Sources/SwiftKit/Switchin.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  Switchin.swift
+//  
+//
+//  Created by Zach Eriksen on 7/7/20.
+//
+
+import Foundation
+
+public func switchin<T: Hashable, O>(value: T,
```

Comment:
This should have doc strings describing the function at a high level and what each param does.